document/core/appendix/embedding.rst

```diff
@@ -693,3 +693,23 @@ Matching
    \F{match\_externtype}(\X{et}_1, \X{et}_2) &=& \TRUE && (\iff \vdashexterntypematch \X{et}_1 \matchesexterntype \X{et}_2) \\
    \F{match\_externtype}(\X{et}_1, \X{et}_2) &=& \FALSE && (\otherwise) \\
    \end{array}
+
+
+.. index:: value type
+.. _embed-default-value:
+
+Value types
+~~~~~~~~~~~
+
+:math:`\F{default\_value}(\type) : \val`
```

Comment:
For consistency with the naming conventions of the other functions, I would probably call this `val_default`, as a sort of value constructor. I'd also suggest to put it in the same section as `ref_type` (and rename the section to "Values").